packages/scm/src/browser/scm-provider.ts

```diff
@@ -64,6 +64,7 @@ export interface ScmResourceDecorations {
 export interface ScmCommand {
     title: string;
     tooltip?: string;
+    id?: string;
```

Comment:
it not it confusing that id is used as command as well? Can we instead fix it in the plugin system, that id is assigned to a command. 

Also `command` and `title` are not optional in vscode.d.ts: https://github.com/microsoft/vscode/blob/6de911e3298424cfb6dd6792bff5499f1cd6a230/src/vs/vscode.d.ts#L19-L29

I wonder why there are optional for us? Can we fix it?

Reply:
> it not it confusing that id is used as command as well? Can we instead fix it in the plugin system, that id is assigned to a command.

done
> Also `command` and `title` are not optional in vscode.d.ts: https://github.com/microsoft/vscode/blob/6de911e3298424cfb6dd6792bff5499f1cd6a230/src/vs/vscode.d.ts#L19-L29
>
>I wonder why there are optional for us? Can we fix it?

The Git extension may set an SCM command item without an action: https://github.com/eclipse-theia/theia/blob/2a978748c6e51a1f31adb34ed4661528e4e98438/packages/git/src/browser/git-contribution.ts#L647-L652